tests/daemon/test-daemon.c

```diff
@@ -252,7 +252,7 @@ append_bytes (GPollableInputStream *pollable_input_stream,
 
   guint new_length = byte_array->len + num_bytes_read;
   g_byte_array_set_size (byte_array, new_length);
-  return num_bytes_read == num_bytes_remaining;
+  return (gsize) num_bytes_read == num_bytes_remaining;
```

Comment:
I always have to look up the precedence of casts versus other operators. I sometimes write:

```suggestion
  return num_bytes_remaining == (gsize) num_bytes_read;
```

but I think this might fall under "if you have been writing C for ten years and still can't remember this simple fact, perhaps this is one of the few cases where C is not at fault" :p